package.json

```diff
@@ -53,6 +53,7 @@
     "build:less:cpy": "copyfiles \"src/**/*.less\" \".\" --up 1 --all",
     "build:less:css": "foreach -x \"lessc #{path} #{dir}/#{name}.css\" -i \"**/*.mixin.less\" -g \"modules/{*,*/*,{draft,esl-utils,esl-forms}/*/*}.less\"",
     "build:docs": "copyfiles \"src/**/*.md\" \".\" --up 1 --all",
+    "build:analyzer": "cd pages && webpack --json ../.report/stats.json && webpack-bundle-analyzer ../.report/stats.json -m static -r ../.report/stats.html",
```

Comment:
It's better to move it under the build-pages category as it does not relate to the main build.